compiler-core/src/parse/error.rs

```diff
@@ -65,7 +65,10 @@ impl ParseError {
             ),
             ParseErrorType::ExprLparStart => (
                 "This parenthesis cannot be understood here",
-                vec!["Hint: To group expressions in gleam use \"{\" and \"}\".".into()],
+                vec![
+                    "Hint: To group expressions in gleam use \"{\" and \"}\".".into(),
+                    "Hint: Tuples are created with `#(` and `)`.".into(),
```

Comment:
Instead of putting "hint" twice, I think it would be better to use one sentence.

Reply:
Agree.
Something like this?

```rust
ParseErrorType::ExprLparStart => (
    "This parenthesis cannot be understood here",
    vec![
        "Hint: To group expressions in Gleam, use \"{\" and \"}\"; tuples are created with `#(` and `)`.".into(),
    ]
),
```

Reply:
Yeah, or just "and"